docs/reference/TensorFlow.md

```diff
@@ -1,405 +1,370 @@
 
-For Windows users, install Ubuntu 20.04 LTS in a VirtualBox. Refer to the following videos to set up:
+    Configure Mbed CLI to find your GNU installation:
 
-**How to install VirtalBox 6.0.10 on Windows 10**
+    ```    
+    mbed config -G GCC_ARM_PATH <path_to_your_arm_toolchain>/bin
+    ```
 
-![IMAGE ALT TEXT HERE](http://img.youtube.com/vi/8mns5yqMfZk/0.jpg)
+* Make 3.82 or newer (if you are using an older version, see the troubleshooting section for instructions).
 
-[Watch here](http://www.youtube.com/watch?v=8mns5yqMfZk)
+    <span class="notes">**Note**: On a Mac, you might have to use gmake instead of make.</span>
 
-**How to install Ubuntu 20.04 on VirtualBox in Windows 10**
+* Python 2.7. We recommend [using pyenv to manage Python versions](https://pypi.org/project/pyenv/).<!--The install page for Mbed CLI now requires 3.7. Why are we asking for 2.7?-->
 
-![IMAGE ALT TEXT HERE](http://img.youtube.com/vi/x5MhydijWmc/0.jpg)
+* Git.
 
-[Watch here](http://www.youtube.com/watch?v=x5MhydijWmc)
+* xxd.<!--unless is exists by default on all Linux computers?-->
 
+* A development board that can run [TensorFlow Lite for Microcontrollers](https://www.tensorflow.org/lite/microcontrollers#supported_platforms). We tested on the [NXP FRDM-K66F](https://os.mbed.com/platforms/FRDM-K66F/) development board.
 
-**Getting started**
+* A mini-USB cable.
 
-TensorFlow Lite for Microcontrollers supports several devices out of the box, and is relatively easy to extend to new devices. For this guide, we will focus on the **NXP FRDM K66F**.
+<!--MarkDown doesn't support embedded YouTube players, which is one reason I removed the videos. Another is that it's better practice to link to official sites, not to videos uploaded by "unofficial" users who may remove them without warning and may, for all we know, be in legal disputes with the companies who created those products (I admit that isn't very likely with "how to install" videos). Also, not sure these videos will work in all countries.-->
+### Download, build and install the application
 
-![IMAGE ALT TEXT HERE](https://raw.githubusercontent.com/COTASPAR/K66F/master/images/nxp_k66f_board.jpeg)
+1. Navigate to the directory where you keep code projects.
+1. Download the TensorFlow Lite source code:
 
+    ```
+    git clone https://github.com/tensorflow/tensorflow.git
+    ```
 
-We will deploy a sample application that uses the microphone on the K66F and a TensorFlow machine learning model to detect the words “yes” and “no”.
+1. Navigate into the project directory:
 
-To do this, we will show you how to complete the following steps:
+    ```
+    cd tensorflow/tensorflow/lite/micro/mbed/
+    ```
+    
+1. Change the DebugLog function to use UnbufferedSerial instead since the Serial API was deprecated in Mbed OS 6:
```

Comment:
Can we upstream this? Or link an open PR in the doc?

Reply:
Many of the other TFLu examples still use older versions of Mbed OS and thus if we upstream it that could break those examples. Ideally, all of the examples would be ported to the latest version of mbed. However, I think for now this should be okay while perhaps we try to engage with the TFLu team to port the other examples.

Reply:
As soon as this is published we will start to work with the TF team to discuss next steps.